### PR TITLE
Use feature detection instead of version detection 

### PR DESCRIPTION
**Pull request by cclauss**
Redo #939

### CODE REVIEW

File: CTFd/utils/__init__.py [hunk 1]
```diff
@@ -28,20 +22,12 @@ def _get_config(key):
         if value and value.isdigit():
             return int(value)
         elif value and isinstance(value, six.string_types):
-            if value.lower() == 'true':
-                return True
-            elif value.lower() == 'false':
-                return False
-            else:
-                return value
+            return {'true': True, 'false', False}.get(value.lower(), value)
```
Comment (ColdHeat) [inline CTFd/utils/__init__.py]: SyntaxError here causing your test failures I believe. I don't see this as more readable, is there some gain to be had here besides the condensed code? If not please revert this change.

File: CTFd/utils/__init__.py [hunk 2]
```diff
@@ -28,20 +22,12 @@ def _get_config(key):
...
 
 
 def get_config(key, default=None):
     value = _get_config(key)
-    if value is None:
-        return default
-    else:
-        return value
+    return default if value is None else value
```
Comment (ColdHeat) [inline CTFd/utils/__init__.py]: This reads okay but I am not entirely sure what the benefit is here besides slightly more condensed code. If there's no benefit besides that please revert.

Comment (cclauss) [reply CTFd/utils/__init__.py]: Would __return value or default__ work in this situation?

**Comment by cclauss** [comment]
Add truth_dict
When we can cut the number of lines of code in half without loss of readability, we should attempt to do so.  Dict lookups should also be encouraged for performance reasons.

**Pull request closed without merging**